ccp/modules/mtba-compose.yml

```diff
@@ -19,18 +19,15 @@ services:
       FILE_CHARSET: ${MTBA_FILE_CHARSET}
       FILE_END_OF_LINE: ${MTBA_FILE_END_OF_LINE}
       CSV_DELIMITER: ${MTBA_CSV_DELIMITER}
-    labels:
-      - "traefik.enable=true"
-      - "traefik.http.routers.mtba.rule=PathPrefix(`/`)"
-      - "traefik.http.services.mtba.loadbalancer.server.port=80"
-      - "traefik.http.routers.mtba.tls=true"
+#    labels:
+#      - "traefik.enable=true"
+#      - "traefik.http.routers.mtba.rule=PathPrefix(`/mtba`)"
+#      - "traefik.http.services.mtba.loadbalancer.server.port=80"
+#      - "traefik.http.routers.mtba.tls=true"
```

Comment:
Why has this been commented out?

Reply:
Currently, we don't need that it is externally reachable. It will be needed when we implement a micro-frontend for the Teiler-UI.